assignments/views.py

```diff
@@ -19,3 +21,18 @@ def get_queryset(self):
 class SectionsViewSet(NestedViewSetMixin, ReadOnlyModelViewSet):
     serializer_class = SectionSerializer
     queryset = Section.objects.filter(assignment__status=Assignment.STATUS_OPEN)
+
+
+class SubmitAnswersViewSet(ViewSet):
+
+    def get_serializer_context(self):
```

Comment:
I wouldn't say ViewSet has get_serializer_context. You assume you are talking about genericviews.